src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content=
+      "width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./stars.css">
+    <link rel="stylesheet" href="./card.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27"
+        class="card__product-image"
+      >
+
+      <p class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <section class="card___stars-box">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star stars_last"></div>
+        </div>
+
+        <div class="card__reviews">
+          Reviews: 5
+        </div>
+      </section>
+
+      <section class="card__price">
+        <div class="card__value-text">
+          Price:
+        </div>
+
+        <div class="card__price-number">
+          $2,199
+        </div>
+      </section>
+
+      <div class="card__button">
+        <a
+          class="card__button-link"
+          href="#"
+          data-qa="hover"
+        >
+          buy
+        </a>
+      </div>
```

Comment:
You can have only <a> tag here, this <div> is not necessary and creates mess in css